recipes/abacas/meta.yaml

```diff
@@ -17,8 +17,10 @@ build:
   noarch: generic
 
 requirements:
+  host:
+    - perl # [not osx]
   run:
-    - perl
+    - perl # [not osx]
```

Comment:
I guess it needs perl on both linux and osx?

Reply:
I've removed the `# [not osx]`, it was a comment dragged form the recipe in another channel.